compiler/dalgona/tests/SingleOperatorTest.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2022 Samsung Electronics Co., Ltd. All Rights Reserved.
```

Comment:
(1) this file name has different convention with other scripts. either way, would be better to follow one style
(2) this has copyright notice, this too would be better to follow one style

Reply:
I changed the file name style (except test.lst for consistency with other tests), and copied copyright notice. PTAL